linchpin/defaults/schemas/schema_v4.json

```diff
@@ -108,8 +108,35 @@
                                  "keyvalue": {
                                      "type": "array"
                                  },
-                                 "hostrequire": {
-                                     "type": "array"
+                                 "hostrequires": {
+                                     "type": "array",
+                                     "items": {
+                                         "not": {
+                                             "allOf": [
+                                                 {"required": ["force", "tag"]}
+                                             ]
+                                         },
```

Comment:
This schema bit is attempting to reflect the mutually exclusive relationship between the use of `force:` under `hostrequires`, and the use of `tag:`, since using `force:` means that any `tag:` filters are ignored by beaker. I have no idea if it works, and would love some pointers on simple ways to actually test this scheme validation against the topologies I've written during the development and testing of this PR.